examples/dicklang/parse.go

```diff
@@ -27,12 +25,7 @@ const (
 )
 
 // Parse parses a dick-lang file
-func Parse(filePath string) (*ModelDicks, error) {
-	// Read the source file into memory
-	bt, err := ioutil.ReadFile(filePath)
-	if err != nil {
-		return nil, err
-	}
+func Parse(fileName, source string) (*ModelDicks, error) {
```

Comment:
we never use `source` as a `string` in `Parse`, instead, we require it to be `[]rune`, therefore `Parse(fileName string, source []rune)` makes more sense.